include/psp2/message_dialog.h

```diff
@@ -0,0 +1,200 @@
+#ifndef _MSG_DIALOG_H_
+#define _MSG_DIALOG_H_
+
+#include "common_api.h"
+#include <psp2/types.h>
+
+enum {
+	//! Illegal parameter
+	SCE_MSG_DIALOG_ERROR_PARAM	= 0x80100a01
+};
+
+enum {
+	//! Max length of a user message
+	SCE_MSG_DIALOG_USER_MSG_SIZE	= 512
+};
+
+typedef SceInt32 SceMsgDialogMode;
+enum {
+	SCE_MSG_DIALOG_MODE_INVALID      = 0,
+	SCE_MSG_DIALOG_MODE_USER_MSG     = 1,
+	SCE_MSG_DIALOG_MODE_SYSTEM_MSG   = 2,
+	SCE_MSG_DIALOG_MODE_ERROR_CODE   = 3,
+	SCE_MSG_DIALOG_MODE_PROGRESS_BAR	= 4
+};
+
+typedef SceInt32 SceMsgDialogSystemMessageType;
+enum {
+	SCE_MSG_DIALOG_SYSMSG_TYPE_INVALID					= 0,
+
+	//! Displays "Please wait."
+	SCE_MSG_DIALOG_SYSMSG_TYPE_WAIT						= 1,
+
+	//! Displays "There is not enough free space on the memory card."
+	SCE_MSG_DIALOG_SYSMSG_TYPE_NOSPACE					= 2,
+
+	//! Displays "Move away from the source of interface, or adjust the compass by moving your PS Vita system as shown below."
```

Comment:
It's a dialog message used with some apps to re calibrate the motion sensors. No idea how that could be used in a homebrew, but I included it anyways